lib/cloudregister/registerutils.py

```diff
@@ -502,6 +502,19 @@ def get_credentials(credentials_file):
     return (username, password)
 
 
+# ----------------------------------------------------------------------------
+def refresh_registry_credentials():
+    """Refresh registry credentials."""
+    # to silence InsecureRequestWarning
+    # should be fixed on a different PR
+    requests.packages.urllib3.disable_warnings()
+    if get_activations():
+        sys.exit(0)
```

Comment:
exiting is the responsibility of the caller. The function shold report appropriate return codes.